src/use_cookie.rs

```diff
@@ -487,7 +488,7 @@ pub struct UseCookieOptions<T, E, D> {
 
     /// Function to add a set cookie header to the response on the server.
     /// When you use one of the features `"axum"` or `"actix"` there's a valid default implementation provided.
-    ssr_set_cookie: Arc<dyn Fn(&Cookie) + Send + Sync>,
+    pub ssr_set_cookie: Arc<dyn Fn(&Cookie) + Send + Sync>,
```

Comment:
Option fields should stay private. They're meant to be used with builder methods.

```suggestion
    ssr_set_cookie: Arc<dyn Fn(&Cookie) + Send + Sync>,
```